addons/tmf_loadouts/macros.inc

```diff
@@ -8,6 +8,8 @@ LIST_1("ACE_tourniquet")
 LIST_2("ACE_bodyBag"), \
 LIST_20("ACE_fieldDressing"), \
 LIST_10("ACE_morphine"), \
+LIST_3("ACE_salineIV_250"), \
+LIST_1("ACE_salineIV_500"), \
 LIST_5("ACE_epinephrine"), \
 LIST_5("ACE_tourniquet"), \
```

Comment:
Drop to 4 tourniquets and add an additional 1 to the rifleman macro per the discussion in discord.